content/docs/04-clusters/02-data-center/01-maas/15-create-manage-maas-clusters.md

```diff
@@ -118,18 +122,28 @@ The cluster status is updated to **Deleting** while cluster resources are being
 
 <br />
 
-## Force Delete a Cluster
 
-If a cluster is stuck in the Deletion state for a minimum of 15 minutes it becomes eligible for force deletion. You can force delete a cluster from the tenant and project admin scopes. To force delete a cluster, follow the same steps outlined in [Delete a MAAS Cluster](/clusters/data-center/maas/create-manage-maas-clusters#deleteamaascluster). After 15 minutes, a **Force Delete Cluster** option is available in the **Settings drop-down menu**. The drop-down menu will provide you with an estimated remaining duration before the force deletion becomes available.
+# Upgrade a MAAS Cluster
 
-<br />
+If you plan to upgrade your MAAS cluster, you should plan and execute it carefully. It helps minimize disruptions to your applications and services and avoid conflicts when managing a MAAS cluster. 
```

Comment:
We should probably tell them to review the upgrade reference page and advice them to take a backup of the cluster.

- https://docs.spectrocloud.com/clusters/cluster-management/backup-restore